Replicator/Pusher.cc

```diff
@@ -301,6 +305,14 @@ namespace litecore::repl {
              && !_options->properties[kC4ReplicatorOptionDisableDeltas].asBool() )
             _deltasOK = true;
 
+            // re; CBL-5680, CBL-5681. Currently replacementRevs only concerns the passive pusher (active puller).
+            // We can enable this code in future to enable the feature on the active pusher.
+#ifdef ACTIVE_PUSH_REPLACEMENTREVS
+        _sendReplacementRevs = reply->boolProperty("sendReplacementRevs"_sl);
+        logInfo("Peer changes response %s request the pusher to send replacementRevs",
+                _sendReplacementRevs ? "did" : "did not");
+#endif
```

Comment:
I feel like this shouldn't be included as when enabling it some tests will fail. We would need a bit more thought about how the checkpoint, remote rev, and checkpoint will work together for active push.

Reply:
I just #ifdef'd it out so we don't lose the code, and when we want the pusher side its an easy change, and we can spend more time fixing the issues rather than trying to remember where the code is supposed to go 😆 . Not intended for it to be actually used until we're ready to go ahead with that.

Reply:
Perhaps a brief comment?

Reply:
I did put a comment above this ifdef already